### PR TITLE
Change the advice about linking to sections and figures

### PR DESCRIPTION
**Pull request by xfq**
To recommend use of [[[ ]]].

### CODE REVIEW

**Comment by himorin** [comment]
I think we already have `[[[#xxx]]]` links (to figures and sections), along with `a` link with classes like `characterClass` or `termref`.
Does this demand to replace all of these with `[[[#xxx]]]`?

**Comment by xfq** [comment]
I think this currently only applies to links to figures and sections. If it's possible to add classes (and customized link text, as I mentioned in https://github.com/w3c/clreq/pull/316#issuecomment-657169127 ) using this syntax, then we should probably use this syntax for all internal links.

File: EDITING.md [hunk 1]
```diff
@@ -119,16 +119,16 @@ Here are some tips on how to maintain the parallel language structure in markup.
     <span class="codepoint" translate="no"><span lang="ja">。</span> [<span class="uname">U+3002 IDEOGRAPHIC FULL STOP</span>]</span>
     ```
 
-- To link to a section, use the Respec feature, ie. link to the id on the section tag and leave the link text empty.
+- To link to a section, use the ReSpec feature, ie. link to the id on the `section` tag and leave the link text empty.
```
Comment (r12a) [inline EDITING.md]: >  and leave the link text empty

I don't think that text is needed any more.  Same applies to the figure instructions.

Comment (xfq) [reply EDITING.md]: Thanks. Updated now.

**Comment by r12a** [comment]
lgtm

**Comment by himorin** [comment]
seems ready to be merged, /cc @kidayasuo

**Review by kidayasuo — APPROVED** [review]
The change looks good. Approving.

**Pull request merged**